tests/contrib/hooks/test_aws_glue_job_hook.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can't we mock this stuff with `moto`? With the current tests, a lot is mocked.

Reply:
Currently, moto does not support glue jobs yet. I opened a feature request (https://github.com/spulec/moto/issues/1561) for this.

I could have contributed but not sure I have much time now.

Reply:
We can use these endpoints right? https://github.com/spulec/moto/pull/1750

I'm ok with merging it right now with the current mock construction, but I think in the future we should move to moto because moto will be closer to the actual calls. Unfortunately we've seen wrongly mocked before.

Reply:
I completely agree. I think that would be my next OSS contribution. 
Thanks